src/ZEO/asyncio/_futures.pyx

```diff
@@ -303,13 +271,6 @@ cdef class CoroutineExecutor:
             if blocking is not None:
                 result._asyncio_future_blocking = False
                 await_next = result
-            elif PY_MAJOR_VERSION < 3 and isinstance(result, asyncio.Future):
-                await_next = result  # trollius predates ._asyncio_future_blocking
-
-            # `yield coro` - handle as if it was `yield from coro`
-            elif _iscoroutine(result):
-                # NOTE - always AsyncTask even if we are originally under ConcurrentTask
-                await_next = AsyncTask(result, self.task.get_loop())
 
             else:
                 # object with __await__ - e.g. @cython.iterable_coroutine used by uvloop
```

Comment:
Why do we need this special `_await__` logic? It is not present in #195 and nevertheless all `uvloop` tests passed.

Reply:
Dieter, thanks for noticing. I simply overlooked to remove this in 7f574ecd. I removed that code in https://github.com/zopefoundation/ZEO/pull/218/commits/32091b5f. Thanks, once again, for pointing this out.